### ISSUE TITLE
Appearance-Navigation - Reset navigation modal appears at the end of reading order [Mac only]

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1147]
Appears to be a bug only present with Safari:

"Description: While testing the appearance-navigation page, it was found that activating the 'reset navigation' button puts the modal at the end of reading order. When modals are activated, focus should move into the modal and remain within the modal until the user dismisses it. Modals should be immediately after the triggering component in the DOM order.

Note: There is a known issue with Safari 26 where VoiceOver focus does not shift properly to the dialog when it opens. A practical workaround is to programmatically move focus inside the dialog after it opens, for example by calling JavaScript focus() on the first interactive element in the dialog or a designated focusable container with tabindex=""0"".

Expected Results: Focus should be taken to the beginning of the modal when a button is pressed to open it.

Actual Results: Focus is not taken to the modal but is still placed on the 'reset navigation' button and the modal is at the end of reading order."

2.4.3 Focus Order; Level A; WCAG 2.0

**Comment by aviannamiller** [comment]
This is not specific to the pattern in Appearance > Navigation. I am seeing focus remain elsewhere in other cases that open dialogs, such as when attempting to delete an item (either from the browse table or an item page).
